### PR TITLE
:seedling: Prompts user to open Okta Verify App after 30 seconds

### PR DESCRIPTION
**Pull request by gowthamidommety-okta**
Resolves: OKTA-127113

screen shot 2017-06-15 at 12 07 21 pm

### CODE REVIEW

**Comment by gowthamidommety-okta** [comment]
@mauriciocastillosilva-okta Please Review

**Comment by gowthamidommety-okta** [comment]
@lboyette-okta Please review

File: test/unit/spec/MfaVerify_spec.js [hunk 1]
```diff
@@ -2011,6 +2011,54 @@ function (Okta,
...
+                Q.stopUnhandledRejectionTracking();
+                return setupFailurePolling(test)
+                .then(function () {
+                  expect(test.form.submitButton().prop('disabled')).toBe(true);
+                  expect(test.form.submitButtonText()).toBe('Push sent!');
+                  expect(test.form.warningMessage()).toBe(
+                    'Haven\'t received a push notification yet? Try opening the Okta Verify App on your phone.');
+                  return tick(test);
+                })
+                .then(function () { return tick(test); })
```
Comment (mauriciocastillosilva-okta) [inline test/unit/spec/MfaVerify_spec.js]: can we see if we can replace this with some ```Expect.waitForError``` or similar?

Comment (gowthamidommety-okta) [inline test/unit/spec/MfaVerify_spec.js]: Thanks! Made changes

**Pull request merged**